### PR TITLE
skipIndexing feature

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [x] Optimization
- [x] Documentation Update

## Description

Ability to skip indexing on certain jobs, useful to prevent capturing datapoints from setup-up jobs

@mohit-sheth I've taken the liberty of adding this flag to the networkPolicy workloads, wdyt?

@josecastillolema I think this might be useful for web-burner as there're some set-up steps in the jobs you probably don't need to capture

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: pkg/prometheus/types.go [hunk 1]
```diff
@@ -43,7 +43,6 @@ type Prometheus struct {
 type Job struct {
 	Start     time.Time
 	End       time.Time
-	Name      string
```
Comment (vishnuchalla) [inline pkg/prometheus/types.go:46]: Can we take this nits in a different PR?

File: pkg/util/metrics/metrics.go [hunk 2]
```diff
@@ -111,7 +111,6 @@ func ProcessMetricsScraperConfig(metricsScraperConfig ScraperConfig) Scraper {
 			p.JobList = []prometheus.Job{{
 				Start: time.Unix(metricsScraperConfig.StartTime, 0),
 				End:   time.Unix(metricsScraperConfig.EndTime, 0),
-				Name:  metricsScraperConfig.JobName,
```
Comment (vishnuchalla) [inline pkg/util/metrics/metrics.go:114]: Looks like this one requires changes at different places as well, from where we are passing this arg.

**Comment by vishnuchalla** [comment]
Do we want to skip [clusterMetadata](https://github.com/cloud-bulldozer/kube-burner/blob/master/pkg/workloads/helpers.go#L261) as well with this flag?

**Comment by rsevilla87** [comment]
> Do we want to skip [clusterMetadata](https://github.com/cloud-bulldozer/kube-burner/blob/master/pkg/workloads/helpers.go#L261) as well with this flag?

If you want to explude cluster's metadata, you should disable indexing :)

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Pull request merged**